app/assets/javascripts/application.js.erb

```diff
@@ -194,9 +194,9 @@ function notificationAlertClose(){
 
 $(document).ready(function(){
   $('.tree-link a').each( function(){
-    truncateLongString( $(this), 30);
+    truncateLongString( $(this), <%= Constants::NAME_TRUNCATION_LENGTH %>);
```

Comment:
I was thinking that you can just assign this ruby constants into a js global variables so we don't need to have those js.erb files around so that the jslint will work again